app/scripts/lib/rpc-method-middleware/handlers/switch-ethereum-chain.js

```diff
@@ -25,11 +25,15 @@ const switchEthereumChain = {
     setProviderType: true,
     setActiveNetwork: true,
     requestUserApproval: true,
+    getNetworkConfigurations: true,
+    getNetworkClientIdForDomain: true,
+    getProviderConfig: true,
   },
 };
+
 export default switchEthereumChain;
 
-function findExistingNetwork(chainId, findNetworkConfigurationBy) {
+export function findExistingNetwork(chainId, findNetworkConfigurationBy) {
```

Comment:
[nit] 😬 😶‍🌫️
Does this still need to be exported?

Reply:
nope! thanks :)
```suggestion
function findExistingNetwork(chainId, findNetworkConfigurationBy) {
```